support-frontend/assets/components/subscriptionCheckouts/subscriptionSubmitButtons.jsx

```diff
@@ -29,8 +34,24 @@ type PropTypes = {|
   billingPeriod: BillingPeriod,
   validateForm: Function,
   formIsValid: Function,
+  allErrors: FormError<FormField>[],
 |};
 
+type ErrorSummaryPropTypes = {
+  errors: Array<Object>,
+}
+
+const ErrorSummary = (props: ErrorSummaryPropTypes) => (
+  <div className="component-form-error__border">
+    <p className="component-form-error__heading">There is a problem</p>
```

Comment:
Should this be a heading tag rather than `<p>`?

Reply:
I have used the custom Heading component here, thanks for the feedback.